elementor/elementor
# Fix: Rename `e-action-hash` attribute to a valid `data-elementor-action-hash` name [ED-9060]

## PR Checklist



## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Change e-action-hash to a valid data attribute data-elementor-action-hash

## Description
An explanation of what is done in this PR

* The [W3C validator](https://validator.w3.org/nu/) shows this error when a lightbox widget(video, gallery) is used:
Attribute e-action-hash not allowed on element div at this point.

## Test instructions
This PR can be tested by following these steps:

1 ) Add video lightbox widget on a page
2 ) check https://validator.w3.org/nu/

## Quality assurance


Fixes #
https://github.com/elementor/elementor/issues/20513
